hazelcast/src/main/java/com/hazelcast/config/DeclarativeConfigUtil.java

```diff
@@ -46,38 +51,38 @@
     /**
      * Array of accepted suffixes for XML configuration files.
      */
-    public static final String[] XML_ACCEPTED_SUFFIXES = new String[]{"xml"};
+    public static final Collection<String> XML_ACCEPTED_SUFFIXES = unmodifiableList(singletonList("xml"));
```

Comment:
Isn't `singletonList()` already immutable?

Reply:
Resolved in https://github.com/hazelcast/hazelcast/pull/14953/commits/8b1fdea661124e48b9b5bdb09c9dfaa48d64319b